src/test/resources/query-tests.json

```diff
@@ -173,8 +173,8 @@
    },
    {
       "query" : "[for (.) number(.)]",
-      "input" : "[\"1\", \"2\", \"3\"]",
-      "output" : "[1,2,3]"
+      "input" : "[\"1\", \"0.1e+302\", \"-0E33\", \"-1E138\", \"1.01E278\", \"1.01e235\", \"-1\", \"-1.0E4\", \"-0.1\"]",
+      "output" : "[1, 1.0E301, -0.0, -1.0E138, 1.01E278, 1.01E235, -1, -10000.0, -0.1]"
```

Comment:
This is actually meant to be a test for array comprehensions, not the number function. So please leave this one unchanged and just make a new test in `function-tests.json`.